webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -3144,6 +3145,117 @@ const AdminDefinition: AdminDefinitionType = {
+                        },
+                        {
+                            type: 'dropdown',
+                            key: 'ServiceSettings.BurnOnReadDurationMinutes',
+                            label: defineMessage({id: 'admin.posts.burnOnRead.duration.title', defaultMessage: 'Burn-on-Read Duration'}),
+                            help_text: defineMessage({id: 'admin.posts.burnOnRead.duration.desc', defaultMessage: 'Sets the countdown duration for Burn-on-Read messages once they are revealed. After a recipient clicks to reveal a BoR message, the message will delete itself for that user after the specified duration. This setting applies to all Burn-on-Read messages.'}),
+                            help_text_markdown: false,
+                            options: [
+                                {
+                                    value: '1',
+                                    display_name: defineMessage({id: 'admin.posts.burnOnRead.duration.1min', defaultMessage: '1 minute'}),
+                                },
+                                {
+                                    value: '5',
+                                    display_name: defineMessage({id: 'admin.posts.burnOnRead.duration.5min', defaultMessage: '5 minutes'}),
+                                },
+                                {
+                                    value: '10',
+                                    display_name: defineMessage({id: 'admin.posts.burnOnRead.duration.10min', defaultMessage: '10 minutes'}),
+                                },
+                                {
+                                    value: '30',
+                                    display_name: defineMessage({id: 'admin.posts.burnOnRead.duration.30min', defaultMessage: '30 minutes'}),
+                                },
+                                {
+                                    value: '60',
+                                    display_name: defineMessage({id: 'admin.posts.burnOnRead.duration.1hour', defaultMessage: '1 hour'}),
+                                },
+                                {
+                                    value: '480',
+                                    display_name: defineMessage({id: 'admin.posts.burnOnRead.duration.8hours', defaultMessage: '8 hours'}),
+                                },
+                            ],
+                            onConfigLoad: (value: any) => value ?? '10',
+                            isDisabled: it.any(
+                                it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.SITE.POSTS)),
+                                it.stateIsFalse('ServiceSettings.EnableBurnOnRead'),
+                            ),
+                            isHidden: it.configIsFalse('FeatureFlags', 'BurnOnRead'),
+                        },
+                        {
+                            type: 'radio',
+                            key: 'ServiceSettings.BurnOnReadAllowedUsers',
```

Comment:
Can we disable this for now? I want to add this once we decide on how to manage if this is going to be this way or another?